includes/wikia/api/UserApiController.class.php

```diff
@@ -35,9 +37,17 @@ public function getDetails() {
 		$ids = array_slice( $ids, 0, static::USER_LIMIT );
 		//users are cached inside the service
 		$users = (new UserService())->getUsers( $ids );
+
 		$items = array();
+
 		foreach ( $users as $user ) {
 			$userName = $user->getName();
+			$powerUserTypes = [];
+			$powerUserTypesRaw = ( new \Wikia\PowerUser\PowerUser( $users[0] ) )->getTypesForUser();
```

Comment:
1. `( new \Wikia\PowerUser\PowerUser( $user ) )->getTypesForUser();` as otherwise you would always send data for first user
2. Can we just return ( new \Wikia\PowerUser\PowerUser( $user) )->getTypesForUser(); as I get it it returns something like:
['type1', 'type2']

I think we don't have to do anything with it